wgsl/index.bs

```diff
@@ -10593,57 +10729,47 @@ struct __modf_result_vecN {
   <thead>
     <tr><th>Parameterization<th>Overload<th>Description
   </thead>
-  <tr algorithm="signed abs">
-    <td>|T| is [SIGNEDINTEGRAL]
-    <td class="nowrap">`abs`(|e|: |T| ) -> |T|
+  <tr algorithm="integral abs">
+    <td>[ALLINTEGRALDECL]
+    <td class="nowrap">`@const fn`<br>`abs`(|e|: |T| ) -> |T|
     <td>The absolute value of |e|.
         [=Component-wise=] when |T| is a vector.
-        If |e| evaluates to the largest negative value, then the result is |e|.
+        If |e| is a signed integral type and evaluates to the largest negative
```

Comment:
.... signed integral **scalar** type...

The general rule for overflow would apply for AbstractInt case